docs/CHANGELOG.md

```
@@ -14,13 +14,25 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added `bu_labels` label type (OSIDB-4982)
 - Auditing and tracking of tool-assisted and auto-created affects (OSIDB-4951, OSIDB-4978)
 - Add UpstreamData model with upstream purl field and collection (OSIDB-4923)
+- document workflow framework
+- introduce EMBARGOED workflow
+- introduce workflow labels
+
+### Changed
+- align existing workflows with workflow framework concepts
+- auto-adjust workflow:state on Flaw.save
 
 ### Changed
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)